distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/TestCatalog.java

```diff
@@ -136,6 +140,76 @@ public void testCswIngest() {
                 hasXPath("//TransactionResponse/InsertResult/BriefRecord/BoundingBox"));
     }
 
+    @Test
+    public void testFilterPlugin() {
+        try {
+            // Ingest the metacard
+            String id1 = ingestXmlFromResource("/metacard1.xml");
+
+            // Test without filtering
+            ValidatableResponse response = executeOpenSearch("xml", "q=*");
+            response.body(hasXPath("/metacards/metacard[@id='" + id1 + "']"));
```

Comment:
I support any effort to reduce magic strings in source code, even in test classes.